plugins/modules/system/lvol.py

```diff
@@ -532,7 +553,10 @@ def main():
             if tool:
                 if resizefs:
                     tool = '%s %s' % (tool, '--resizefs')
-                cmd = "%s %s -%s %s%s %s/%s %s" % (tool, test_opt, size_opt, size, size_unit, vg, this_lv['name'], pvs)
+                if size_operator:
+                    cmd = "%s %s -%s %s%s%s %s/%s %s" % (tool, test_opt, size_opt, size_operator, size, size_unit, vg, this_lv['name'], pvs)
```

Comment:
I see `size_operator` being passed as-is to the command, and it seems it can only have the value `+`, but I suppose we should have the `-` case as well. Is that something for a future PR or just missing from this one?

Reply:
And as far as I have looked (not all that much I must confess), there is no support for `-` when using `%` either. That could be a good idea for this PR or a next one.

Reply:
You're right. Since I'm already changing the code myself I will add that option as well. 
As for the `%`; the `+|-` are supported but only with lvreduce and lvextend. lvcreate has no mention of either `+|-`. They should only be used when resizing existing volumes.